.obs/chartfile/operator/values.yaml

```diff
@@ -9,11 +9,6 @@ seedImage:
   tag: "%VERSION%"
   imagePullPolicy: IfNotPresent
 
-channel:
```

Comment:
This is still supported if users want to bring in (or are already bringing in) a channel from a value. We just removed the default.

Reply:
![Screenshot from 2024-07-29 13-08-29](https://github.com/user-attachments/assets/f91dc095-f2b9-4876-b0e1-c119a10c3d9d)